system_files/etc/profile.d/brew.sh

```diff
@@ -1,2 +1,12 @@
 #!/usr/bin/env bash
-[[ -d /home/linuxbrew/.linuxbrew && $- == *i* ]] && eval "$(/home/linuxbrew/.linuxbrew/bin/brew shellenv)"
+if [[ -d /home/linuxbrew/.linuxbrew && $- == *i* ]] ; then
+  export HOMEBREW_PREFIX="/home/linuxbrew/.linuxbrew"
+  export HOMEBREW_CELLAR="/home/linuxbrew/.linuxbrew/Cellar"
+  export HOMEBREW_REPOSITORY="/home/linuxbrew/.linuxbrew/Homebrew"
+  [ -z "${MANPATH-}" ] || export MANPATH=":${MANPATH#:}"
+  export INFOPATH="/home/linuxbrew/.linuxbrew/share/info:${INFOPATH:-}"
```

Comment:
```suggestion
  export INFOPATH="${INFOPATH}:/home/linuxbrew/.linuxbrew/share/info"
```

I feel like we should deprioritize brew on the documentation side as well, would be weird if you get the manpage for brew git even tho `git` points to `/usr/bin/git`